src/files/file-input/FileInput.js

```diff
@@ -65,6 +66,15 @@ export default class FileInput extends React.Component {
     })
   }
 
+  componentDidUpdate (prev) {
+    if (this.props.addProgress === 100 && prev.addProgress !== 100) {
+      this.setState({ force100: true })
```

Comment:
Can you talk me through the issue that means `force100` is needed? Could we just drop the last call to `updater(null)` from the downloader and avoid the need for this?

Reply:
We use `force100` so the transition from Finished to Add Files is not immediate and takes 2 seconds. The same for downloads. We don't want to change Finished to Download immediately or the user won't even see it.